.github/release-drafter.yml

```diff
@@ -0,0 +1,22 @@
+# Format and labels used aim to match those used by Ansible project
```

Comment:
It looks like it's copied from Ansible, can we update the template in accordance with the current one. See [6.0.0](https://github.com/jazzband/pip-tools/releases/tag/6.0.0) release for example.

Reply:
From ansible-lint likely...

Reply:
Oh, [that](https://github.com/ansible-community/ansible-lint/releases) template looks pretty similar to ours.

Reply:
@atugushev Please just push all desired changes directly to that branch, it. would make it reach acceptable status much faster this way.

I found the list of currently configured labels bit confusing as I was not sure which ones end-up in which place in changelog. One thing that needs to be decided first is which is the one-of list. for sure we need one as `skip-changelog` for stuff that we do not want to mention in changelog bug for others we need a clean list.

The other label-check action does assure that we do not merge by mistake a change that is missing any-of the  required labels. Sadly there is no standardization in that area.